mautic/mautic
# Tags on edit should display same message as other entities, for e.g. xyz has been updated!

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Part 1:
Alert text and behaviour is different than other entities in the application
General message is Entity is updated. In case of tags it say tag already present.

Part 2:
User is unable to clone tag with same text. Application should give meaningful warning or clone functionality shouldn't be present for tags. So, it is better to remove the clone option as this has no significant purpose with tag, which has only 1 field. 



#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new Tag with some text
3. Click on Save button
4. Click on Save and Close button

After PR:
1. The message is '<TAG_NAME> has been updated!'.
2. The clone option is not visible.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10929"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
